network/src/peer_manager/builder.rs

```diff
@@ -354,8 +354,8 @@ impl PeerManagerBuilder {
             pm_context.connection_reqs_rx,
             pm_context.upstream_handlers,
             pm_context.connection_event_handlers,
-            pm_context.max_concurrent_network_reqs,
```

Comment:
Great catch, Do we have any idea what was the side effect of this? Were we dropping network messages because of misconfigured channel size?

Reply:
my gut feeling is it doesn't matter much, those channels are either keyed by peer id or one per peer